core/include/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_subdirectory(gz/plugin)
+install(DIRECTORY ignition DESTINATION ${IGN_INCLUDE_INSTALL_DIR_FULL})
```

Comment:
I think this file can be removed as well (look at the last 2 lines of `core/CMakeLists.txt` just above)